Nimble/DSL+Wait.swift

```diff
@@ -38,13 +38,22 @@ internal class NMBWait: NSObject {
 /// Wait asynchronously until the done closure is called.
 ///
 /// This will advance the run loop.
-public func waitUntil(timeout timeout: NSTimeInterval, file: String = __FILE__, line: UInt = __LINE__, action: (() -> Void) -> Void) -> Void {
-    NMBWait.until(timeout: timeout, file: file, line: line, action: action)
+public func waitUntil(timeout timeout: NSTimeInterval, file: String = __FILE__, line: UInt = __LINE__, action: ((Any?...) -> Void) -> Void) -> Void {
```

Comment:
Also I noticed in the code duplication. The only different is default parameter. Would it be better to replace two of these implementations with the following:
```
public func waitUntil(timeout timeout: NSTimeInterval? = 1, file: String = __FILE__, line: UInt = __LINE__, action: ((Any?...) -> Void) -> Void) -> Void {
    NMBWait.until(timeout: timeout!, file: file, line: line, action: bridgeToVoidAction(action))
}
```